jans-cli-tui/cli_tui/plugins/010_auth_server/ssa.py

```
@@ -625,10 +629,15 @@ def ssa_details(self, **params: Any) -> None:
     def show_token(self, dialog):
         async def coroutine():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `coroutine`

Add return type annotation: `None`

(ANN202)